server/controllers/router.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+const error = require("./error");
+
+router.get("/", (req, res) => {
```

Comment:
This works for setup but would abstract this out once you started working. Why are you sending an index page that doesn't exist? 🙈